### ISSUE TITLE
Permission denied error with Docker Extension

### ISSUE THREAD

**Issue by jcornick-alabs** [telepresenceio/telepresence#3159]
When attempting to connect to a cluster, an MacOS dialog about network connections and error in Telepresence Extension:

```jsx
connector.Connect: error loading config file "/root/.cache/telepresence/kube/arn:aws:eks:us-west-2:978296336269:cluster/demo-eks": open /root/.cache/telepresence/kube/arn:aws:eks:us-west-2:978296336269:cluster/demo-eks: permission denied
```

Despite clicking “Allow” in the MacOS dialog, the connection attempt fails. The Telepresence error is repeated for all subsequent connection attempts, but no MacOS dialog is repeated.

**Comment by jcornick-alabs** [comment]
@jacoblbeck was this fixed in the last release?

**Comment by jacoblbeck** [comment]
It potentially could have, but we weren't give reproducing steps and i couldn't reproduce it with our eks clusters.

**Comment by knlambert** [comment]
May have been fixed with 2.13.2. On hold until we get a reproducer.

**Comment by srmxjp** [comment]
Still present on `v2.14.2`
